source/troubleshooting.rst

```diff
@@ -24,11 +24,11 @@ Default logging levels for the server can be altered by setting logging environm
 - DJANGO_LOGGING_LEVEL
   - Infrastructure logging
 
-If your system does not have SELinux enabled, you can start the Quipucords server with the following Docker command with increased logging::
+If your system does not have SELinux enabled and you are installing release 1.0.0, you can start the Quipucords server with the following Docker command with increased logging::
```

Comment:
If your system does not have SELinux enabled, you can start the Quipucords server with the following Docker command with increased logging. In the following example, the installed version of Quipucords is 1.0.0::

Reply:
👍